src/scripts/DataLayer.js

```diff
@@ -9,10 +9,8 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
-/* eslint no-console: "off" */
-const mergeWith = require('lodash/mergeWith');
-const cloneDeep = require('lodash/cloneDeep');
-const get = require('lodash/get');
+
+const _ = require('lodash');
```

Comment:
I would rather we include dependencies separately as it usually results in a smaller final size.

Reply:
Addressed in https://github.com/adobe/adobe-client-data-layer/pull/66/commits/2c1e47cb98050221d8a5b112c389b68be41a0229